src/main/webapp/css/main.css

```diff
@@ -78,9 +82,36 @@ button {
   overflow-y: scroll;
 }
 
+/*The following will probably not be needed.*/
```

Comment:
Let's just remove all the commented out code. If we need it back, we can get it from source control. :)